pkgs/by-name/dh/dhcpig/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "dhc-pig";
+  version = "1.6";
+  pyproject = true;
```

Comment:
```suggestion
```
This doesn't have a `pyproject.toml`. I'm pretty sure that we can remove this.

Reply:
This should stay as it's defines the build system. We are moving towards PEP-517.